bridge/scripts/mint-script.ts

```diff
@@ -0,0 +1,286 @@
+import { ethers } from "ethers";
+import EthersAdapter from "@safe-global/safe-ethers-lib";
+import SafeServiceClient from "@safe-global/safe-service-client";
+import Safe from "@safe-global/safe-core-sdk";
+import {
+    SafeTransaction,
+    SafeTransactionDataPartial,
+} from "@safe-global/safe-core-sdk-types";
+import { AwsKmsSigner } from "ethers-aws-kms-signer";
+import { Command } from "commander";
+import readline from "readline";
+
+const provider = new ethers.providers.JsonRpcProvider(process.env.RPC_URL);
+
+// Create a wallet instance from the sender's private key
+const owner1Signer = new AwsKmsSigner(
+    {
+        accessKeyId: process.env.SAFE_OWNER_1_AWS_ACCESS_KEY_ID,
+        secretAccessKey: process.env.SAFE_OWNER_1_AWS_SECRET_ACCESS_KEY,
+        region: process.env.SAFE_OWNER_1_AWS_REGION!,
+        keyId: process.env.SAFE_OWNER_1_AWS_KEY_ID!,
+    },
+    provider
+);
+const owner2Signer = new AwsKmsSigner(
+    {
+        accessKeyId: process.env.SAFE_OWNER_2_AWS_ACCESS_KEY_ID,
+        secretAccessKey: process.env.SAFE_OWNER_2_AWS_SECRET_ACCESS_KEY,
+        region: process.env.SAFE_OWNER_2_AWS_REGION!,
+        keyId: process.env.SAFE_OWNER_2_AWS_KEY_ID!,
+    },
+    provider
+);
+const owner3Signer = new AwsKmsSigner(
+    {
+        accessKeyId: process.env.SAFE_OWNER_3_AWS_ACCESS_KEY_ID,
+        secretAccessKey: process.env.SAFE_OWNER_3_AWS_SECRET_ACCESS_KEY,
+        region: process.env.SAFE_OWNER_3_AWS_REGION!,
+        keyId: process.env.SAFE_OWNER_3_AWS_KEY_ID!,
+    },
+    provider
+);
+
+const ethAdapterOwner1 = new EthersAdapter({
+    ethers,
+    signerOrProvider: owner1Signer,
+});
+
+const txServiceUrl = process.env.SAFE_TX_SERVICE_URL!;
+const safeService = new SafeServiceClient({
+    txServiceUrl,
+    ethAdapter: ethAdapterOwner1,
+});
+let safeSdkOwner1: Safe;
+let safeAddress: string;
+
+// If you have an existing Safe, you can use it instead of deploying a new one
+// https://app.safe.global/home?safe=eth:0x4A2FbE06004e37dE6Fe7Da59a53D14a407Def0ed
```

Comment:
I think it's not a good idea to keep this comment, (Why ETH Mainnet Safe Address here?) could be better to just remove it.

Reply:
removed comments which include contract address
fixed!